CHANGELOG.md

```diff
@@ -75,6 +75,8 @@
 - Added `onChannelAvatarClick` handler to `MessageListHeader`. [#4545](https://github.com/GetStream/stream-chat-android/pull/4545)
 
 ### ⚠️ Changed
+- 🚨 Breaking change: Renamed `onHeaderActionClick` to `onHeaderTitleClick` in `MessagesScreen`. Change made in order to better reflect the handler's behavior. [#4535](https://github.com/GetStream/stream-chat-android/pull/4535)
```

Comment:
Please like this comment,  resolve and merge after checking the following commit: https://github.com/GetStream/stream-chat-android/pull/4535/commits/a1fa3e85d720fa7aef01f827991c6cc40eb745b8